pktgen/app/pktgen-tool.c

```diff
@@ -418,6 +421,40 @@ void stop_pkt_frwding(int sig __rte_unused) {
   is_first_tx = true;
 }
 
+/* Function: start_gtp_traffic
+ * Input param: None
+ * Output param: None
+ * Return values: None
+ */
+void start_gtp_traffic(int sig __rte_unused) {
```

Comment:
it composes the traffic with gtp header or just send normal traffic and go through gtp tunnel?

Reply:
Depends on the command line argument (If the command used to start il_trafficgen was '-g', it starts sending tunneled traffic. If the command to start il_trafficgen was '-r', it starts sending normal traffic.) This is the cli version of what you would have done in the tmux screen: "start 0".